server/src/main/antlr4/org/eclipse/lsp/cobol/core/parser/CobolPreprocessorLexer.g4

```diff
@@ -22,6 +22,14 @@ lexer grammar CobolPreprocessorLexer;
         lastTokenText = token.getText();
     }
 }
+// compiler directive tokens
+TITLESTATEMENT : (T I T L E ' '+ .*? NEWLINE)  ;
+
+CONTROL_DIRECTIVE: ASTERISKCHAR (C O N T R O L | C B L) ((' '| COMMACHAR)
```

Comment:
I see the list of directives is incomplete, is it intentional?

Reply:
Only mentioned the tokens which were a combination of multiple identifiers. Mentioned these avoid mismatch in preprocessor and cobol lexer.

Reply:
Also, why do you write `C B L` this way when it is declared explicitly? I would recommend declaring also the `CONTROL`

Reply:
done.